tests/test_resolver.py

```diff
@@ -26,3 +50,99 @@ def test_open_stage_and_logging(capfd):
     assert "_OpenAsset" in outputs[6]
     assert "_GetModificationTimestamp" in outputs[7]
     assert "_GetExtension" in outputs[8]
+
+
+# As all the data tends to follow the same form, convenience method
+# to avoid repeating myself
+def assert_parking_lot_structure(usd_stage):
+    floor = usd_stage.GetPrimAtPath("/ParkingLot/ParkingLot_Floor_1")
+    car1 = usd_stage.GetPrimAtPath("/ParkingLot/ParkingLot_Floor_1/Car1")
+    car2 = floor.GetChild("Car2")
+
+    assert car1.GetPropertyNames() == ["color", "spec_color"]
+    assert car2.GetPropertyNames() == ["color", "spec_color"]
+
+
+# Given a USD document that references an asset via a direct relative
+# file path, then the asset is resolved to the file path as expected.
+def test_openassetio_resolver_has_no_effect_with_no_search_path():
+
+    stage = Usd.Stage.Open(
+        "resources/integration_test_data/resolver_has_no_effect_with_no_search_path/parking_lot.usd"
+    )
+
+    assert_parking_lot_structure(stage)
+
+
+# Given a USD document that references an asset via a relative search-path
+# based file path, then the asset is resolved the the file path as expected.
+def test_openassetio_resolver_has_no_effect_with_search_path():
+
+    context = build_search_path_context(
+        "resources/integration_test_data/resolver_has_no_effect_with_search_path/search_path_root"
+    )
+    stage = Usd.Stage.Open(
+        "resources/integration_test_data/resolver_has_no_effect_with_no_search_path/parking_lot.usd",
+        context,
+    )
+
+    assert_parking_lot_structure(stage)
+
+
+# Given a USD document that recursively references multiple other USD
```

Comment:
[fix] I struggled to grok this for a bit Can we use 'assetized' here instead of just 'resolves/references' - as resolve is a USD term ie. everything is being resolved by `Ar` even if its a file path. Might be worth updating the folder names to be consistent too?

I mention this not to nit-pick, but this wording is defining the expected behaviour so needs to be unambiguous in the future about path based references and assetized ones. How you have it for the child/grandchild scenarios seems perfect.

Reply:
I'm not sure if this is what you're getting at, to my eye these are equally clear, so I'm prone to have misinterpreted you.

https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/13/commits/22dce9219328fd7f3b375d57ca189a8aa66225b2

Reply:
Perfect, thanks, it was the missing 'assetized' bit - as all things are resolved...